src/main/java/forestry/storage/features/BackpackItems.java

```diff
@@ -14,6 +14,7 @@
 public class BackpackItems {
 	private static final IFeatureRegistry REGISTRY = ModFeatureRegistry.get(Constants.MOD_ID).getRegistry(ForestryModuleUids.BACKPACKS);
 
+	//TODO there is a race condition here: If Backpacks loads before apiculture then the group is null.
```

Comment:
I'm not 100% on the cause but there's definitely something breaking due to `ItemGroups` with these recipes.

Reply:
This should be fixed now.

Reply:
The problem still exists. If I put breakpoints on the static initialiser in `ItemGroupForestry` and `BackpackItems` then `BackpackItems` is loaded first. But I'm not really sure how to solve this without making the ItemGroups static initialisers, which means we can't hide items when the module isn't enabled.

Reply:
The easiest fix would be to give @FeatureProvider a priority value. And priorities ItemGroupForestry.

Reply:
https://github.com/ForestryMC/ForestryMC/blob/f02df303c2bbd66ab13ef8c212747a845c423755/src/main/java/forestry/modules/CommonModuleHandler.java#L99-L100

Reply:
It works perfectly now, thanks.